code/modules/mob/living/living_defines.dm

```diff
@@ -120,3 +120,15 @@
 	var/slowed_by_drag = TRUE //Whether the mob is slowed down when dragging another prone mob
 
 	var/is_busy = FALSE //Used for random actions that take time. ex: curbstomping. We need to make sure we can only do one of these at a time.
+
+	var/obj/chattext/chattext = new
+	
+	var/last_heard_name = null
+	var/last_used_color = null

```

Comment:
Woah! Would you look at this!
https://github.com/goonstation/goonstation/blob/ac7bd846a4163a046b1ca4e0ae8f4636258825f5/code/mob/living.dm#L85-L87